packages/sanity/src/core/releases/store/createReleaseOperationStore.ts

```diff
@@ -144,23 +194,31 @@ export function createReleaseOperationsStore(options: {
     } as IdentifiedSanityDocumentStub
 
     await (IS_CREATE_VERSION_ACTION_SUPPORTED
-      ? requestAction(client, [
-          {
-            actionType: 'sanity.action.document.createVersion',
-            releaseId,
-            attributes: versionDocument,
-          },
-        ])
-      : client.create(versionDocument))
+      ? requestAction(
+          client,
+          [
+            {
+              actionType: 'sanity.action.document.createVersion',
+              releaseId,
+              attributes: versionDocument,
+            },
+          ],
+          opts,
+        )
+      : client.create(versionDocument, opts))
```

Comment:
Frankly a bit unsure about this, but it worked (when not using the dry run) so this should be fine